js/index.js

```diff
@@ -24,19 +35,66 @@ function calculateAll() {
 
 function removeProduct(event) {
   const target = event.currentTarget;
-  console.log('The target in remove is:', target);
+  const parent = target.parentNode.parentNode.parentNode;
+  parent.removeChild(target.parentNode.parentNode)
+  calculateAll()
+}
   //... your code goes here
+/*
+function testRow() {
+  const newRow = document.getElementById('cart');
+  newRow.insertRow(-1)
+  console.log(newRow)
 }
+testRow()*/
 
 // ITERATION 5
 
-function createProduct() {
-  //... your code goes here
+function createProduct(event) {
+  const targetName = document.querySelector('.create-product').querySelector('input[type="text"]')
+  let nameValue = targetName.value;
```

Comment:
i would have probably suggest to add a class to that field so that you can access it more easily.
But if you would like to leave HTML as it is, then instead of chaining querySelectors you could access it like this:
`  const targetName = document.querySelector('.create-product > td > input[type=text]').value`